submission/submission-security/src/main/kotlin/ebi/ac/uk/security/integration/components/ISecurityService.kt

```diff
@@ -6,10 +6,12 @@ import ebi.ac.uk.api.security.LoginRequest
 import ebi.ac.uk.api.security.RegisterRequest
 import ebi.ac.uk.api.security.ResetPasswordRequest
 import ebi.ac.uk.api.security.RetryActivationRequest
+import ebi.ac.uk.model.MigrateHomeOptions
 import ebi.ac.uk.model.User
 import ebi.ac.uk.security.integration.model.api.SecurityUser
 import ebi.ac.uk.security.integration.model.api.UserInfo
 
+@Suppress("TooManyFunctions")
```

Comment:
I've noticed we have a lot of these. Should we consider incrementing the number in the config instead of suppressing it everywhere? 🤔 

It looks like our design methods need more functions 😁

Reply:
make sense